dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2025 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0"  
+                   xmlns:ibm="http://www.ibm.com/xmlns/appservers/osgi/metatype/v1.0.0"
+                   xmlns:ibmui="http://www.ibm.com/xmlns/appservers/osgi/metatype/ui/v1.0.0"
+                   localization="OSGI-INF/l10n/metatype">
+
+    <OCD id="io.openliberty.microprofile.health" name="%mpHealth" description="%mpHealth.desc" ibm:alias="mpHealth" ibm:beta="true">
+        <!--  filePeriodSecond defaults to 10 seconds, but we do not specify default here because we'll programmatically do it. 
+        So we can throw warning if they set this, but havn't enabled file checking -->
+        <AD id="fileUpdateInterval" name="%fileUpdateInterval" description="%fileUpdateInterval.desc" required="false" type="String" ibm:beta="true"/>
```

Comment:
Can't you add the `ibm:type="duration(ms)" default="10s"` to this, for better handling of error scenarios, instead of programmatically re-inventing the logic?

Reply:
The presence of the configuration attribute enables the functionality.
A default value can not be discernable from a user provided 10s.

Reply:
Won't the metatype server config, handle the cases, for user errors, if they typed something like "abcs", it would automatically throw an error, instead of us explicitly handling that? Since, we set the meta type to `ibm:type="duration(ms)"?